### ISSUE TITLE
Request for an event

### ISSUE THREAD

**Issue by icoderharshit** [vinitshahdeo/Hacktoberfest2021#400]
### Please provide the following information

- **Is this event for college students or professionals?** College students

- **How many folks do you expect will attend this event?** 300-400
- **When do you want this event?** 03-10-2021

- **Language of the event?** English

- [x] fill [this](https://forms.gle/7FQtAnFVJESSFXJ3A) google form.

**Comment by vanshika-srivastava** [comment]
@icoderharshit Please mention the requirements of event

**Comment by icoderharshit** [comment]
## Mentors: 1-2
## Day & Time (may change in the worst case): October 3, 2021 4:00 P.M - 5:30 P.M
## Required resources for the Presentation (e.g. PPT)

**Comment by vanshika-srivastava** [comment]
alright , what exactly do you want the speakers to talk about ? Beginner friendly issues , finding projects ,contribution to open source , etc  ?

Mention that and I will reach out to you by tomorrow

**Comment by icoderharshit** [comment]
Yes, on how to Find projects and beginner friendly issues & in general a bit of why contributing to open-source community and how.
